server/src/test/java/com/broadcom/lsp/cobol/usecases/engine/UseCaseEngine.java

```diff
@@ -209,10 +210,17 @@ private static void assertResultEquals(AnalysisResult actual, TestData expected)
     assertResult("Constant usages:", expected.getConstantUsages(), actual.getConstantUsages());
 
     assertResult(
-        "Paragraph definition:",
-        expected.getParagraphDefinitions(),
-        actual.getParagraphDefinitions());
-    assertResult("Paragraph usage:", expected.getParagraphUsages(), actual.getParagraphUsages());
+        "Paragraph definition:", expected.getParagraphDefinitions(), actual.getParagraphDefinitions());
+
+    if (!actual.getParagraphUsages().equals(actual.getSectionUsages())) {
```

Comment:
This if condition, itself can be asserted, so test continues

Reply:
Sounds reasonable, but it should be a part of a different activity

Reply:
I mean no need if condition, just replacing it with asserted line ;)

Reply:
It will not work because when we have PERFORM statement it is a paragraph usage and section usage at the same time. But expected paragraph usages is not equal to actual paragraph usages in that case :-)

Reply:
I have refactored this statement, maybe it is better now

Reply:
Refactored due to change the logic